src/FFDHE.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_FFDHE_H_
+#define MUMBLE_FFDHE_H_
+
+/// FFDHE provides access to the Diffie-Hellman parameters from RFC 7919.
+class FFDHE {
+	public:
+		/// PEMForNamedGroup returns the PEM-encoded
+		/// Diffie-Hellman parameters for the RFC 7919
+		/// group with the given name, such as "ffdhe2048",
+		/// "ffdhe4096", etc.
+		///
+		/// Returns an empty and null byte array if the request
```

Comment:
Yes it is. From the docs:

> For historical reasons, QByteArray distinguishes between a null byte array and an empty byte array. A null byte array is a byte array that is initialized using QByteArray's default constructor or by passing (const char *)0 to the constructor. An empty byte array is any byte array with size 0. A null byte array is always empty, but an empty byte array isn't necessarily null:

    QByteArray().isNull();          // returns true
    QByteArray().isEmpty();         // returns true
    
    QByteArray("").isNull();        // returns false
    QByteArray("").isEmpty();       // returns true
    
    QByteArray("abc").isNull();     // returns false
    QByteArray("abc").isEmpty();    // returns false

> All functions except isNull() treat null byte arrays the same as empty byte arrays. For example, data() returns a pointer to a '\0' character for a null byte array (not a null pointer), and QByteArray() compares equal to QByteArray(""). We recommend that you always use isEmpty() and avoid isNull().